vcd/resource_vcd_catalog_item.go

```diff
@@ -187,23 +193,10 @@ func createOrUpdateCatalogItemMetadata(d *schema.ResourceData, meta interface{})
 
 	log.Printf("[TRACE] adding/updating metadata for catalog item")
 
-	vcdClient := meta.(*VCDClient)
-
-	adminOrg, err := vcdClient.GetAdminOrgFromResource(d)
-	if err != nil {
-		return fmt.Errorf(errorRetrievingOrg, err)
-	}
-
-	catalog, err := adminOrg.GetCatalogByName(d.Get("catalog").(string), false)
-	if err != nil {
-		log.Printf("[DEBUG] Unable to find catalog: %s", err)
-		return nil
-	}
-
-	catalogItem, err := catalog.GetCatalogItemByName(d.Get("name").(string), false)
```

Comment:
Please tell more why this code became not needed after this PR?

Reply:
Just removed repeatable code which was in already existing function